src/App.tsx

```diff
@@ -1,26 +1,28 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { useState } from 'react';
 import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import { Authenthication } from './pages/Authenthication';
+import { Invoices } from './pages/Invoices';
+import { AuthenticationPrivateRoutes } from './components/form-authentication-components/AuthenticationPrivateRoutes';
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.tsx</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+
+	const [currentTheme, setCurrentTheme] = useState(() => {
```

Comment:
odpowiadajac na pytanie- to nie jest ok. Powinien zrobic themeContext lub wlasnego hooka w folderze hooks,ktorego nazwalbys useTheme (to lepsza opcja)

Reply:
zrobiony usecontext